README.md

```diff
@@ -27,9 +27,18 @@ To use this dbt package, you must have the Fivetran **SAP** (sap.com) the respec
 - bseg
 - faglflexa
 - faglflext
+- kna1
```

Comment:
Can the step 1 section be rephrased a bit. It sounds a bit strange when reading it over. Additionally, can we link the SAP connector that is used so customers can know exactly where they need to ensure these tables are being synced.

Reminder to also check with Edwin what the proper link for the connector is. I believe it is the HVA connector, but would prefer to confirm with him.

Reply:
> https://fivetran.com/docs/databases/sap-erp/high-volume-agent/hva-sap-ecc-hana

Reply:
Given Edwin's answer, would something like:

> from [Fivetran's connectors like LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements), [HVA SAP ECC](https://fivetran.com/docs/databases/sap-erp/high-volume-agent/hva-sap-ecc-hana) or [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) 

work?

Reply:
Yup that is great!

Reply:
On the same note as the transform package, I would recommend reworking this section to be something like the following.

```md
## Step 1: Prerequisites
To use this dbt package, you must have the following:
- At least one Fivetran of the following SAP connectors:
   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
   - [HVA SAP ECC](https://fivetran.com/docs/databases/sap-erp/high-volume-agent/hva-sap-ecc-hana)
   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) 
-  Within the connector, syncing the following respective tables into your destination:
  - bkpf
  - bseg
  - faglflexa
  - faglflext
  - kna1
  - lfa1
  - mara
  - pa0000
  - pa0001
  - pa0007
  - pa0008
  - pa0031
  - ska1
  - t001
  - t503
  - t880
- A **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, **Databricks** destination.
```

Reply:
Main point here is to have this section be consistent between both package READMEs.

Reply:
Updated